cloud/scope/mocks/cluster_client_mock.go

```diff
@@ -36,171 +36,185 @@ func (m *MockClusterScopeClient) EXPECT() *MockClusterScopeClientMockRecorder {
 }
 
 // DeleteApiServerLB mocks base method.
-func (m *MockClusterScopeClient) DeleteApiServerLB(ctx context.Context) error {
+func (m *MockClusterScopeClient) DeleteApiServerLB(arg0 context.Context) error {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "DeleteApiServerLB", ctx)
+	ret := m.ctrl.Call(m, "DeleteApiServerLB", arg0)
```

Comment:
Is there a reason this is changed to `arg0`? `ctx` in Go is pretty common for `context`

Reply:
I didn't notice that since I used the gomock tool to generate this file. The command I used :
```
mockgen -package=mock_scope github.com/oracle/cluster-api-provider-oci/cloud/scope ClusterScopeClient > cluster_client_mock.go
```
I can update this function by using ctx.

Reply:
Ah if the tool does it don't worry at all 😄